packages/auth/src/grant/routes.ts

```diff
@@ -309,9 +344,14 @@ async function handleGrantChoice(
   ctx.status = 202
 }
 
+interface FinishParams {
+  id: string
+  nonce: string
+}
```

Comment:
Maybe `StartParams` and `FinishParams` could be consolidated into a single interface